cmd/rpcfuzz/argfuzz/argfuzz.go

```diff
@@ -4,12 +4,13 @@
 package argfuzz
 
 import (
+	cryptorand "crypto/rand"
 	"encoding/hex"
 	"math/rand"
```

Comment:
tempted to just replace with `"crypto/rand"`, but maybe for another day

Reply:
I'm not sure I understand why we're replacing math with crypto

Reply:
it's for the ci stuff when i updated it to go v1.20. 

`rand.Read` and `rand.Seed` is deprecated and will error. https://github.com/maticnetwork/polygon-cli/actions/runs/5525842651/jobs/10079899036